### DISCUSSION TITLE
HELP, `clear-defaults=true` also disabled my customized keybindings!

### DISCUSSION THREAD

**Discussion by liozur** [zellij-org/zellij#3084] [Q&A] [answered]
I'm moving from `tmux` to `zellij` and trying to disable `zellij`'s default key bindings to build my own from scratch. From the documentation, I note that `clear-defaults=true` can be used to disable `zellij`'s default key bindings. However, when this setting is used, `zellij` is then unable to read my own defined key bindings, for example:

```
keybinds clear-defaults=true {
    resize {
        // not working!
        bind "Ctrl z" { SwitchToMode "Normal"; }
        bind "i" { Resize "Increase"; }
        bind "d" { Resize "Decrease"; }
    }
}
```

What should I do? Do I need to write the custom key bindings in another file? Or is there something I'm not noticing?

Thanks a lot!

**Comment by Lockszmith-GH** [comment]
Ctrl z - that's the default key binding for "suspend and send job to background" - it just might be a bad option.

Have you tried something different?

Can you share your entire config file?

**Reply by liozur** [reply]
Thank you for your reply.

Actually, it doesn't work when I use other keybindings. The above is just an example.

My config file is almost identical to the default one, the only difference is that I `used clear-defaults=true` and added some key bindings as shown below.

By the way, commenting instead of deleting is just easier for `yyp` :)

[Zellij Config.txt](https://github.com/zellij-org/zellij/files/14045678/Zellij.Config.txt)

**Reply by Lockszmith-GH** [reply, marked as answer]
the default state is `normal`, you have commented everything out of it, including the command that enters you in to 'Resize mode':

```
shared_except "resize" "locked" {
        // bind "Ctrl n" { SwitchToMode "Resize"; }
    }
```

Uncomment the `Ctrl-n` line and you should have the resize working.

**Reply by liozur** [reply]
That's it, thanks for describing it in a succinct way :)

**Comment by liozur** [comment]
Solved. It seems like I misunderstood something.

**Reply by liozur** [reply]
I still haven't figured out the whole issue. But at least my config file is working. Maybe I'll come back here after I've figured it all out to give a brief explanation of the problem.
